service/src/main/java/bio/terra/workspace/common/utils/FlightUtils.java

```diff
@@ -4,20 +4,25 @@
 import bio.terra.stairway.FlightContext;
 import bio.terra.stairway.FlightMap;
 import bio.terra.stairway.FlightState;
+import bio.terra.stairway.Stairway;
+import bio.terra.stairway.exception.FlightWaitTimedOutException;
 import bio.terra.workspace.generated.model.ApiErrorReport;
 import bio.terra.workspace.service.iam.AuthenticatedUserRequest;
 import bio.terra.workspace.service.job.JobMapKeys;
 import bio.terra.workspace.service.workspace.exceptions.MissingRequiredFieldsException;
 import bio.terra.workspace.service.workspace.model.Workspace;
+import java.time.Duration;
+import java.time.Instant;
 import java.util.Map;
 import java.util.Optional;
+import java.util.concurrent.TimeUnit;
 import javax.annotation.Nullable;
 import org.springframework.http.HttpStatus;
 
 /** Common methods for building flights */
 public final class FlightUtils {
 
-  public static final int FLIGHT_POLL_SECONDS = 10;
+  public static final int FLIGHT_POLL_SECONDS = 1;
```

Comment:
A bunch of places uses FLIGHT_POLL_SECONDS and FLIGHT_POLL_CYCLES. If you decrease FLIGHT_POLL_SECONDS, you should increase FLIGHT_POLL_CYCLES.

Reply:
Could you leave these and just use a different number for the one flight you care about?

Reply:
The polling waits the whole interval for the first go-round, and thus every sub-flight was taking a minimum of ten seconds. These add up during clone, which uses many sub-flights, so I wanted to keep this change.